odoo/addons/base/views/res_company_views.xml

```diff
@@ -7,7 +7,7 @@
             <field name="arch" type="xml">
                 <form string="Company">
                   <sheet>
-                    <field name="logo" widget="image"  class="oe_avatar"/>
+                    <field name="logo" widget="image" class="oe_avatar" placeholder="/base/static/img/company_image.png"/>
```

Comment:
Value in model was `res_company_logo.png` instead, is there a difference?

Reply:
Same issue with not using the web/image route